test/system/700-play.bats

```diff
@@ -988,52 +988,94 @@ _EOF
 }
 
 @test "podman play with automount volume" {
-    imgname="automount-test-$(safename)"
-    podname="p-$(safename)"
-    ctrname="c-$(safename)"
-
-    cat >$PODMAN_TMPDIR/Containerfile <<EOF
+    imgname1="automount-test-$(safename)1"
+    imgname2="automount-test-$(safename)2"
+    podname_cmd="p-$(safename)-cmd"
+    ctrname_cmd="c-$(safename)-cmd"
+    podname_file="p-$(safename)-file"
+    ctrname_file="c-$(safename)-file"
```

Comment:
When I see a variable of the form `xxx_file`, my brain thinks "this references a file path". Even after staring at this code for half an hour, I still can't get used to the fact that this means "pod and container that have something to do with a file". Could you think of variable names that better reflect the nature of the test being run?